src/include/brain/operating_unit.h

```diff
@@ -116,11 +164,26 @@ class ExecutionOperatingUnitFeature {
    */
   size_t GetNumKeys() const { return num_keys_; }
 
+  /**
+   * @returns estimated cardinality as a reference
+   */
+  size_t &GetCardinality() { return cardinality_; }
+
   /**
    * @returns estimated cardinality
    */
   size_t GetCardinality() const { return cardinality_; }
 
+  /**
+   * @returns num concurrent as a reference
```

Comment:
return not returns

Reply:
wait, we return a reference? if this is to modify, consider making it use a setter instead?

CLion doesn't seem to pick up any users of this..

Reply:
ditto comments for the other getters above in this file